services/core-web/src/components/mine/Summary/MineWorkInformation.js

```diff
@@ -0,0 +1,279 @@
+import React, { Component } from "react";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+import { Row, Col, Divider, Button, Descriptions, List, Popconfirm, Typography, Badge } from "antd";
+import { EditOutlined, DeleteOutlined } from "@ant-design/icons";
+import { openModal, closeModal } from "@common/actions/modalActions";
+import {
+  fetchMineWorkInformations,
+  createMineWorkInformation,
+  updateMineWorkInformation,
+  deleteMineWorkInformation,
+} from "@common/actionCreators/workInformationActionCreator";
+import * as Strings from "@common/constants/strings";
+import { getMineWorkInformations } from "@common/selectors/workInformationSelectors";
+import AddButton from "@/components/common/AddButton";
+import AuthorizationWrapper from "@/components/common/wrappers/AuthorizationWrapper";
+import * as Permission from "@/constants/permissions";
+import { formatDate, formatDateTime } from "@common/utils/helpers";
+import { isEmpty } from "lodash";
+import { modalConfig } from "@/components/modalContent/config";
+import { CoreTooltip } from "@/components/common/CoreTooltip";
+import { getWorkInformationBadgeStatusType } from "@/constants/theme";
+
+const propTypes = {
+  mineGuid: PropTypes.string.isRequired,
+  mineWorkInformations: PropTypes.objectOf(PropTypes.any).isRequired,
+  fetchMineWorkInformations: PropTypes.func.isRequired,
+  createMineWorkInformation: PropTypes.func.isRequired,
+  updateMineWorkInformation: PropTypes.func.isRequired,
+  deleteMineWorkInformation: PropTypes.func.isRequired,
+  openModal: PropTypes.func.isRequired,
+  closeModal: PropTypes.func.isRequired,
+};
+
+const defaultProps = {};
+
+const { Text } = Typography;
+
```

Comment:
Is this syntax passing unit tests? The antd4 updates had issues with referencing components this way

Reply:
NO it wasn't. Fixed